Robot Code/src/org/usfirst/frc/team3316/robot/commands/DBugCommandGroup.java

```diff
@@ -11,8 +11,8 @@
  */
 public abstract class DBugCommandGroup extends CommandGroup 
 {
-	DBugLogger logger = Robot.logger;
-	Config config = Robot.config;
+	protected DBugLogger logger = Robot.logger;
+	protected Config config = Robot.config;
```

Comment:
You need to be within a static method in order to refer to static vaiables. It's not C++.